build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
 	implementation "net.fabricmc.unpick:unpick:$project.unpick_version"
 	implementation "net.fabricmc.unpick:unpick-format-utils:$project.unpick_version"
 	implementation "net.fabricmc:tiny-mappings-parser:$project.tiny_mappings_parser_version"
+	implementation "com.google.code.gson:gson:$project.gson_version"
```

Comment:
I'd rather use something else since Gson [is not actively maintained anymore](https://github.com/google/gson/issues/1821), but I guess this works too. I can switch this to something else in a later PR if you want to.

Reply:
What would your suggestion be? Im happy to change it to something else.

Reply:
Why not Groovy's JsonSlurper?

Reply:
This isnt groovy, and ideally id like to find a replacement that we can use across projects.